tests/potential/test_pair.py

```diff
@@ -0,0 +1,147 @@
+"""Unit tests for pair module."""
+import unittest
+
+import numpy as np
+
+from relentless import core
+from relentless.potential import pair
+from relentless.potential import potential
+
+class test_LennardJones(unittest.TestCase):
+    """Unit tests for pair.LennardJones"""
+
+    def test_init(self):
+        """Test creation from data"""
+        lj = pair.LennardJones(types=('1','2'))
+        coeff = potential.CoefficientMatrix(types=('1','2'), params=('epsilon','sigma','rmin','rmax','shift'),
+                                            default={'rmin':False,'rmax':False,'shift':False})
+        self.assertCountEqual(lj.coeff.types, coeff.types)
+        self.assertCountEqual(lj.coeff.params, coeff.params)
+        self.assertDictEqual(lj.coeff.default.todict(), coeff.default.todict())
+
+    def test_accessors(self):
+        """Test accessor methods for coeffs"""
+        lj = pair.LennardJones(types=('1','2'))
+
+        #test update with kwargs
+        lj.coeff['1','1'].update(epsilon=2.0, sigma=1.5)
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertEqual(lj.coeff['1','2']['epsilon'], None)
+        self.assertEqual(lj.coeff['1','2']['sigma'], None)
+
+        #test update with positional arg
+        lj.coeff['1','2'].update({'epsilon':2.5})
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertAlmostEqual(lj.coeff['1','2']['epsilon'], 2.5)
+        self.assertEqual(lj.coeff['1','2']['sigma'], None)
+
+        #test setitem with default reset
+        lj.coeff['1','2'] = {'sigma':0.5}
+        self.assertAlmostEqual(lj.coeff['1','1']['epsilon'], 2.0)
+        self.assertAlmostEqual(lj.coeff['1','1']['sigma'], 1.5)
+        self.assertEqual(lj.coeff['1','2']['epsilon'], None)
+        self.assertAlmostEqual(lj.coeff['1','2']['sigma'], 0.5)
+
+    def test_energy(self):
+        """Test _energy method"""
+        lj = pair.LennardJones(types=('1','2'))
+
+        lj.coeff['1','1'].update(epsilon=2.0, sigma=1.5)
+        s11 = lj.coeff['1','1']['sigma']
+        e11 = lj.coeff['1','1']['epsilon']
+
+        lj.coeff['1','2'].update(epsilon=1.0, sigma=0.5)
+        s12 = lj.coeff['1','2']['sigma']
+        e12 = lj.coeff['1','2']['epsilon']
+
+        #test scalar r
+        r_input = 11
+        u_actual = -0.00005144
```

Comment:
I like this test for single `r` input, but the values you are checking are really small. The LJ potential is basically zero for `r > 3*sigma`. You might want to test a smaller r so that the energy is numerically significant.